object-cache.php

```diff
@@ -419,7 +460,7 @@ function key($key, $group) {
 		else
 			$prefix = $this->blog_prefix;
 
-		return preg_replace('/\s+/', '', WP_CACHE_KEY_SALT . "$prefix$group:$key");
+		return preg_replace('/\s+/', '', WP_CACHE_KEY_SALT . "_$prefix$group:$key");
```

Comment:
Can we use `:` as the sep here instead please? Keeping consistent with the rest of the key.